### DISCUSSION TITLE
no way to use mouse to get to game menu (escape menu)

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#9711] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

the only way to get to the escape/game menu is with keyboard.

while not exactly a bug, it is a surprising bit of inaccessibility that most every game/program i can think of has.

### Bug prevalence

Happens every time I play

### Version

0.18.15.0

### Which operating system did you encounter this bug on?

Windows

**Comment by MaaiG** [comment]
This is usually a strategy game, some RPG or some casual type... you know.

Some games (such as simulators or first-person games, etc) do prevent this possibility because want to avoid overloading the interface. Especially the elements influencing the immersion in the game. Do you think this is appropriate in this game? You play without a keyboard?
Can you play this game without a keyboard?!
Where do you suggest to put the exit button?
Maybe you can just assign the `ESC` button to one of the mouse buttons?

Honestly, I wish there were as few buttons on the screen as possible.  ;)

**Comment by NilanthAnimosus** [comment]
I do have a keyboard that uses batteries (comically) so I think I understand why this may be wanted, but I also sort of agree with MaaiG here that this is sort of a game that fundamentally requires a keyboard, and although we cannot rebind the escape menu key...I'm not sure if a game like barotrauma strictly requires this unless your in dedicated fullscreen with your keyboard suddenly dying and no other screen to click on (As that's how I usually get out of that).

The game doesn't strictly have good support for gamepad's or steam deck (And would be very awkward to play on either of them really) so I'd still call this a keyboard game for the moment.

Perhaps a button to the main menu is not a bad idea in the above case to avoid being locked into the game until a round ends to disconnect when your keyboard dies, its likely very niche of an issue. it does mean slightly more UI clutter, but there is also some space or ways to perhaps hide such a button as well.

**Comment by MaaiG** [comment]
@Tsunder , @NilanthAnimosus 
Duct tape:
If mouse does not have additional buttons and (or) support for programmable macros you can use third-party software.
As a temporary solution to the problem, you may want something like this: [X-Mouse Button Control (Portable)](https://www.highrez.co.uk/downloads/XMouseButtonControl.htm) - 9mb.
For some reason it works with a delay and a second time... but it works like "Esc"
![Image 001_15-02](https://user-images.githubusercontent.com/93348499/181502171-0a43df5b-7352-4b40-befc-f297acfe5007.png)

**Comment by NilanthAnimosus** [comment]
Although external software to give additional functionality to certain combinations of mouse buttons / key presses or to rebind the escape key in general, the average user wanting this should probably not be expected to learn and do this.
